action.yml

```diff
@@ -43,53 +43,54 @@ runs:
+    - name: 'Check if Go version is a stable release'
+      id: check-stable
       shell: bash
       run: |
         GO_VERSION=${{ steps.get-go-version.outputs.go-version }}
-        GO_MINOR_VERSION=${GO_VERSION%.*}
+        GO_MAJOR_VERSION=${{ steps.get-go-version.outputs.go-major-version }}
         versions=$(curl -s 'https://api.github.com/repos/golang/go/git/refs/tags' | jq -r '.[].ref' | grep -o 'refs/tags/go[0-9\.]*' | grep -E 'go[0-9]+\.[0-9]+\.[1-9][0-9]*$' | sed 's_refs/tags/__')
         latest_versions=$(echo "$versions" | grep -o 'go[0-9]\+\.[0-9]\+' | sed 's/go//' | sort -uV | tail -2)
         
-        lts=false
+        stable=false
         needs_bump=false
         for version in $latest_versions; do
           echo "$versions" | grep "$version"
-          if [ "$version" = "$GO_MINOR_VERSION" ]; then
-            echo "$GO_VERSION is part of an LTS release of $GO_MINOR_VERSION"
-            lts=true
-            echo "lts=${lts}" >> "${GITHUB_OUTPUT}"
+          if [ "$version" = "$GO_MAJOR_VERSION" ]; then
+            echo "$GO_VERSION is a minor release of $GO_MAJOR_VERSION"
+            stable=true
+            echo "stable=${stable}" >> "${GITHUB_OUTPUT}"
             break
           fi
         done
-        if [ "$lts" = "false" ]; then
-          echo "$GO_VERSION is NOT part of an LTS release of $GO_MINOR_VERSION"
+        if [ "$stable" = "false" ]; then
+          echo "$GO_VERSION is NOT a minor release of $GO_MAJOR_VERSION"
           exit 1
         fi
 
-        latest_patch_version=$(echo "$versions" | grep "$GO_MINOR_VERSION"| sed 's/go//' | sort -V | tail -1)
+        latest_patch_version=$(echo "$versions" | grep "$GO_MAJOR_VERSION"| sed 's/go//' | sort -V | tail -1)
         if [ "$latest_patch_version" != "$GO_VERSION" ]; then
-          echo "$GO_VERSION needs to be bumped to $latest_patch_version"
+          echo "$GO_VERSION can be bumped to $latest_patch_version"
           needs_bump=true
           echo "needs-bump=${needs_bump}" >> "${GITHUB_OUTPUT}"
           echo "latest-patch-version=${latest_patch_version}" >> "${GITHUB_OUTPUT}"
         fi
 
-    - name: 'Push changes to bump go version and sdlibs'
+    - name: 'Push changes to bump go version and stdlibs'
       id: git-push-changes
       shell: bash
-      if: steps.check-lts.outputs.lts == 'true' && steps.check-lts.outputs.needs-bump == 'true'
+      if: steps.check-stable.outputs.stable == 'true' && steps.check-stable.outputs.needs-bump == 'true'
       run: |
-        echo "Starting to Push changes to bump go version and sdlibs"
+        echo "Starting to push changes to bump go version and sdlibs"
         git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
         git config --local user.name "github-actions[bot]"  
 
-        BRANCH_NAME="bump-go-${{ steps.check-lts.outputs.latest-patch-version }}"
+        BRANCH_NAME="bump-go/${{ steps.check-stable.outputs.latest-patch-version }}"
```

Comment:
~ Tiny suggested change here that I included for brevity: Dependabot uses `dependabot/` as a prefix in its PRs, and "bump" is common enough a word that this might help avoid manual PR collisions.